legal-api/src/legal_api/services/filings/validations/alteration.py

```diff
@@ -0,0 +1,70 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Validation for the Alteration filing."""
+from http import HTTPStatus
+from typing import Dict
+
+from flask_babel import _
+
+from legal_api.errors import Error
+from legal_api.models import Business
+
+from ... import namex
+from ...utils import get_str
+
+
+def validate(business: Business, filing: Dict) -> Error:
+    """Validate the Alteration filing."""
+    if not business or not filing:
+        return Error(HTTPStatus.BAD_REQUEST, [{'error': _('A valid business and filing are required.')}])
+    msg = []
+
+    nr_path = '/filing/alteration/nameRequest/nrNumber'
```

Comment:
constants are CAPITALIZED
or, as we're moving towards more typing, use 
```
from typing import Final
NR_PATH: Final = '/filing/alteration/nameRequest/nrNumber'
```
or
```
from typing import Final
NR_PATH: Final[str] = '/filing/alteration/nameRequest/nrNumber'
```

Reply:
@thorwolpert I get a linting error when I use caps
`src/legal_api/services/filings/validations/alteration.py:33:4: C0103: Variable name "NR_PATH" doesn't conform to snake_case naming style (invalid-name)`